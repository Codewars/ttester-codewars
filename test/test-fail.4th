\ Copyright 2019 nomennescio
<{ 0 -> 1 }>
