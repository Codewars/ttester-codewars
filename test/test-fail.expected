
<FAILED::>Expected 1 , got 0 
